### ISSUE TITLE
test-suite fails in `subprojects/libnkutils/tests/bindings`

### ISSUE THREAD

**Issue by davidnorthetal** [DaveDavenport/rofi#827]
On latest arch-linux:
`Linux myname 4.17.2-1-ARCH #1 SMP PREEMPT Sat Jun 16 11:08:59 UTC 2018 x86_64 GNU/Linux`

building `rofi-git`:
https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=rofi-git
```
======================================
   rofi 1.5.1-dev: ./test-suite.log
======================================

# TOTAL: 17
# PASS:  16
# SKIP:  0
# XFAIL: 0
# FAIL:  1
# XPASS: 0
# ERROR: 0

.. contents:: :depth: 2

FAIL: subprojects/libnkutils/tests/bindings
===========================================

/nkutils/bindings/press/key/found: OK
/nkutils/bindings/press/key/not-found/modified: OK
/nkutils/bindings/press/combo/minus: OK
/nkutils/bindings/press/combo/plus: OK
/nkutils/bindings/press/combo/gtk: OK
/nkutils/bindings/press/combo/multi-modifier: OK
/nkutils/bindings/press/consume/consumed: OK
/nkutils/bindings/press/consume/not-consumed: OK
/nkutils/bindings/press/keycode/not-consumed: OK
/nkutils/bindings/press/keycode/consumed: OK
/nkutils/bindings/release/key/exclamation: OK
/nkutils/bindings/release/key/gtk: OK
/nkutils/bindings/release/combo/simple: OK
/nkutils/bindings/release/combo/multi-modifiers: OK
/nkutils/bindings/press-and-release/key: OK
/nkutils/bindings/compose/with-binding: **
ERROR:subprojects/libnkutils/tests/bindings.c:517:_nk_bindings_tests_func: assertion failed (fixture->triggered == key->triggered): (1 == 0)
**
ERROR:subprojects/libnkutils/tests/bindings.c:516:_nk_bindings_tests_func: assertion failed (key->text == text): ("`" == NULL)
**
ERROR:subprojects/libnkutils/tests/bindings.c:517:_nk_bindings_tests_func: assertion failed (fixture->triggered == key->triggered): (1 == 0)
FAIL
/nkutils/bindings/compose/without-binding: **
ERROR:subprojects/libnkutils/tests/bindings.c:516:_nk_bindings_tests_func: assertion failed (key->text == text): ("`" == NULL)
FAIL
FAIL subprojects/libnkutils/tests/bindings.test (exit status: 1)
```

**Comment by sardemff7** [comment]
Do you have the file `/usr/share/X11/locale/en_US.UTF-8/Compose`?
Do you have a custom `~/.XCompose` file?

**Comment by davidnorthetal** [comment]
I had a `~/.XCompose` file, but wasn't really using it, so I removed it and the tests-passed.

**Comment by sardemff7** [comment]
I wish `makepkg` had proper env cleaning. :-)
I’ll push something for that later.

**Comment by sardemff7** [comment]
Pushed a fix in 50265f244bc96a553f81c228222dcc9d5c48ace6.
